### PR TITLE
Adds a small note describing BUILD_opencv_world

### LINKED ISSUE #12252: Feature Request "World Library"
OpenCV => Git Head, checked out yesterday, hash: 76eb38976103c86610f73d87e46b7cb5a74f0017
Operating System / Platform => Windows 64 Bit
Compiler => Visual Studio 2017

Windows Developer documentation, link here:

https://docs.opencv.org/master/dd/d6e/tutorial_windows_visual_studio_opencv.html

Suggests there is a "World" library that is a combination of all libraries, specifically the phrase used is:

> Alternatively, your OpenCV download may have been built into one large .lib file. Check by looking in OpenCV\build\architecture\vc14\lib.

Observation: Current CMAKE configuration does not produce anything like a "world library"

My request is to create a "world library" as suggested in the documentation.

Or please remove this suggestion from the documentation.

### PR DESCRIPTION
**Pull request by HarshDolhare**
Fixes  #12252

Adds small documentation clarifying BUILD_opencv_world

### CODE REVIEW

File: doc/tutorials/introduction/windows_install/windows_install.markdown [hunk 1]
```diff
@@ -319,6 +319,7 @@ libraries). If you do not need the support for some of these, you can just freel
         you are concerned about performance, build them and run.
     -   *BUILD_opencv_python* -\> Self-explanatory. Create the binaries to use OpenCV from the
         Python language.
+    -   *BUILD_opencv_world* -\> With this your OpenCV download can be built into one large .lib file.
```
Comment (alalek) [inline doc/tutorials/introduction/windows_install/windows_install.markdown]: What is about this?

> Build OpenCV modules as one large "opencv_world" binary.

File: doc/tutorials/introduction/windows_install/windows_install.markdown [hunk 2]
```diff
@@ -318,7 +318,8 @@ libraries). If you do not need the support for some of these, you can just freel
     -   *BUILD_PERF_TESTS* -\> There are also performance tests for many OpenCV functions. If
         you are concerned about performance, build them and run.
     -   *BUILD_opencv_python* -\> Self-explanatory. Create the binaries to use OpenCV from the
-        Python language.
+        Python language.
```
Comment (mshabunin) [inline doc/tutorials/introduction/windows_install/windows_install.markdown]: Please, remove trailing whitespace on this line.

File: doc/tutorials/introduction/windows_install/windows_install.markdown [hunk 3]
```diff
@@ -318,7 +318,8 @@ libraries). If you do not need the support for some of these, you can just freel
     -   *BUILD_PERF_TESTS* -\> There are also performance tests for many OpenCV functions. If
         you are concerned about performance, build them and run.
     -   *BUILD_opencv_python* -\> Self-explanatory. Create the binaries to use OpenCV from the
-        Python language.
+        Python language. 
+    -   *BUILD_opencv_world* -\> Builds OpenCV module as one large "opencv_world" binary.With this your OpenCV download can be built into one large .lib or .dll file.
```
Comment (mshabunin) [inline doc/tutorials/introduction/windows_install/windows_install.markdown]: The second sentence is not necessary, it gives same information as the first one.

**Comment by HarshDolhare** [comment]
I want more small bugs to solve. is there anything for me?

**Comment by alalek** [comment]
If you are using Python, you could take this one: #12349

**Comment by HarshDolhare** [comment]
hey 
i want some more bugs to solve. is there something i can solve?

**Pull request merged**